### DISCUSSION TITLE
Error Messages

### DISCUSSION THREAD

**Discussion by briandea** [lucky-media/landtamic#29] [General]
I am a new Statamic user and having a difficult time with the Landtamic starter kit.  When changing logo's error message the given data was invalid displays.  It won't allow an upload of SVG or ICO files.  Also, Collections/Pages when modifying the Home page any of the text then saving gets error message:  Method Illuminate\Validation\Validator::validateUniqueEntryValue does not exist.

**Comment by lokmanm** [comment]
Hey @briandea, the error is because of Statamic 5 and we haven't released a version compatible with v5.

We will release an update ASAP, but until then I posted here a temporary fix:
https://github.com/lucky-media/landtamic/issues/28#issuecomment-2112426284
